invenio_administration/assets/semantic-ui/js/invenio_administration/actions/Actions.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Delete } from "./Delete";
+
+export class Actions extends Component {
+  render() {
+    const {
+      displayEdit,
+      displayDelete,
+      actions,
+      apiEndpoint,
+      resource,
+      successCallback,
+      idKeyPath,
+    } = this.props;
+
+    // if number of actions is greater than 3, we display all in a dropdown
+    const displayAsDropdown =
```

Comment:
(from CI): this is unused by now, is that on purpose?

Reply:
yes, to be needed later

Reply:
would be nice to have a `TODO` there. Helped me a lot to visualize missing parts when the module was firstly created.